File: twilio/twilio-api-scripts/stream.py
# twilio helper library
from twilio.rest import Client

# other imports
import time
import requests
import json
import os
import uuid

# your account sid and auth token from twilio.com/console
account_sid = os.environ['TWILIO_ACCOUNT_SID']
auth_token = os.environ['TWILIO_AUTH_TOKEN']
# the twilio client
client = Client(account_sid, auth_token)
# make the outgoing call
call = client.calls.create(
  twiml = '<Response><Start><Stream url="wss://url.to.deepgram.twilio.proxy" track="both_tracks" /></Start><Dial>+11231231234</Dial></Response>', # replace number with person B, replace url
  to = '+11231231234', # person A
  from_ = '+11231231234' # your twilio number
)

File: twilio/node/twilio-api-scripts/stream.js
// Twilio helper library
const twilio = require("twilio");

// Your account SID and auth token from twilio.com/console
const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;

// The Twilio client
const client = twilio(accountSid, authToken);

// Make the outgoing call
client.calls
  .create({
    twiml:
      '<Response><Start><Stream url="wss://url.to.deepgram.twilio.proxy" track="both_tracks" /></Start><Dial>+11231231234</Dial></Response>', // replace number with person B, replace url
    to: "+11231231234", // person A
    from: "+11231231234", // your Twilio number
  })
  .then((call) => console.log(call.sid))
  .catch((err) => console.error(err));
